server/tests/backends/fixtures/filter/dates.json

```diff
@@ -0,0 +1,74 @@
+{
+  "exclude": [
+    "sql"
+  ],
+  "step": {
+    "pipeline": [
+      {
+        "name": "filter",
+        "condition": {
+          "column": "CREATED",
+          "operator": "gt",
+          "value": "2020-01-01 01:03:30"
```

Comment:
hm.... how do we know this will be translated as datetime.datetime when we read this json in the tests?

Reply:
We know that because of the type of the column here :  https://github.com/ToucanToco/weaverbird/blob/67d2f0b802694615912c0237af26fde761c3301e/server/tests/backends/fixtures/filter/dates.json#L30 , if we change that value to a none datetime string the fixture will failed and we should have this kind of error : 
![Screenshot from 2021-11-18 17-47-15](https://user-images.githubusercontent.com/22576758/142459350-2c8b5e8f-40ef-4ddc-bd14-13f451c2b1b9.png)

Reply:
That's true for the data, but does that also work when we read the pipeline?